CONTRIBUTING.md

```diff
@@ -149,6 +149,8 @@ Here is the object for The Movie Database as an example:
 
 Make sure the icon is added in alphabetical order. If you're in doubt, you can always run `npm run our-lint` - this will tell you if any of the JSON data is in the wrong order.
 
+The filename of the SVG should correspond to the `title` above. In general, it should be all lowercase and whitespace and special characters must be removed (full details can be found [in this snippet](https://github.com/simple-icons/simple-icons/blob/36abec3c39887814155c59ff9c7f7d04ba41b044/scripts/utils.js)). If you're in doubt, you can always run `npm run get-filename -- Brand name` to get the correct filename.
```

Comment:
I think the file naming convention isn't clear in CONTRIBUTING.md, and it's also mentions in an irrelevant section.

Maybe use a seperate section called "SVG File Naming Convention" or add it to [Check The Icon section](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#5-check-the-icon)?

Also mentioning the rules may come in handy for those who don't clone the repo, _only fork it and do a quick change via GitHub_.

Here's a Suggestion for the SVG File Naming Convention section written in markdown: _(this section will be below the Update JSON data section)_

---


## SVG File Naming Convention

The filename of the SVG should correspond to the **title** used in the JSON file mentioned above, and it should follow these rules: _(full details can be found [in this snippet](https://github.com/simple-icons/simple-icons/blob/36abec3c39887814155c59ff9c7f7d04ba41b044/scripts/utils.js))_

1. Use **lowercase** letters **without whitespace**

    ```yml
    title: Adobe Photoshop
    filename: adobephotoshop.svg
    ```

1. Only use **latin** letters:

    ```yml
    title: Citroën
    filename: citroen.svg
    ```

1. Replace the following symbols with their alias depending on their position in the title:

    | Symbol | Start | Middle | End  |
    | :----: | ----- | ------ | ---- |
    |   +    | plus  | plus   | plus |
    |   .    | dot-  | -dot-  | -dot |
    |   &    | and-  | -and-  | -and |

    ```yml
    title: .Net
    filename: dot-net.svg
    ```

If you're in doubt, you can always run this script to get the correct filename:

```node
npm run get-filename -- Brand name
```

Reply:
Thanks a bunch for the feedback @YoussefRaafatNasry, I wasn't really sure how or where to write it so I just drafted something quick in hopes of getting some feedback and this looks fantastic! I will update this PR according to your suggestions 😃